buildkite/run_check_deploy_docker_build.sh

```diff
@@ -4,20 +4,10 @@
 # built successfully. This check is done in the buildkite-assets repo to prevent
 # an unbuildable Dockerfile from being deployed to buildkite instances.
 
-DOCKER_IMAGE_NAME=check-heroku-deploy-image
+echo "--- Building and testing with panorama_command.sh"
+HEROKU_DEPLOY_PATH=deploy/heroku-deploy \
+  HEROKU_DEPLOY_IMAGE_NAME=check-heroku-deploy-image \
+  DEPLOYMENT_APP_NAME=panorama-addons \
+  bash deploy/panorama_command.sh custom
```

Comment:
`panorama_command.sh custom` [indicates](https://github.com/panorama-ed/buildkite-assets/blob/main/deploy/panorama_command.sh#L41) we're using a custom command, as opposed to `deploy` and `db_migrate` pre-set commands. The second argument would be the custom command to run, but we don't actually need to provide one; without it, the [script](https://github.com/panorama-ed/buildkite-assets/blob/main/deploy/heroku-deploy/heroku_deploy.sh#L28) just [prints `-- NO COMMAND PASSED IN --`](https://buildkite.com/panorama-education/buildkite-assets/builds/306#0184d03f-b73b-478b-acf9-9f3f8fa61e41/197-201). All we're trying to do is verify the container script works, so that's good enough for us.